src/components/Layout/Sidebar/SidebarLink.tsx

```diff
@@ -66,7 +66,7 @@ export function SidebarLink({
           'text-primary dark:text-primary-dark': level === 0 && !selected,
           'text-base text-secondary dark:text-secondary-dark':
             level > 0 && !selected,
-          'text-base text-link dark:text-link-dark bg-highlight dark:bg-highlight-dark border-blue-40 hover:bg-highlight hover:text-link dark:hover:bg-highlight-dark dark:hover:text-link-dark':
+          'text-base text-primary dark:text-link-dark bg-highlight dark:bg-highlight-dark border-blue-40 hover:bg-highlight hover:text-link dark:hover:bg-highlight-dark dark:hover:text-link-dark':
```

Comment:
Instead of using `text-primary` can we add `test-link-secondary` with value `#045975` (blue-60). This will keep the vibe, while maintaining a good contrast ratio.